docs/best-practices/continuous-integration.md

```diff
@@ -483,7 +483,7 @@ In your app settings on Nevercode, navigate to the **Environment** tab to manage
 
 ## Run tests
 
-Testing in Nevercode with _fastlane_ is 100% automatic. By default, [`scan`](https://docs.fastlane.tools/actions/scan/) is used to invoke the test run, but you can easily swap it out for any other `lane` that you have configured for your test runs. Test results are automatically collected and you don't need to do anything to convert the results from one format to another or place them in a special location on the build machine.
+Testing in Nevercode with **_fastlane_ for iOS** is 100% automatic. By default, [`scan`](https://docs.fastlane.tools/actions/scan/) is used to invoke the test run, but you can easily swap it out for any other `lane` that you have configured for your test runs. Test results are automatically collected and you don't need to do anything to convert the results from one format to another or place them in a special location on the build machine.
```

Comment:
Could we make "iOS" and "Android" subheadlines (level `###`) here for better structuring? The text can stay the same, but the bolding can be removed then.

Reply:
Good point!